### ISSUE TITLE
S6421 FP for Azure Functions with already wrapped body in try/catch block. 

### ISSUE THREAD

**Issue by stefansta1** [SonarSource/sonar-dotnet#5995]
### Description

A violation of the rule "csharpsquid:S6421 (Azure Functions should use Structured Error Handling)" is shown even if a try catch block is used in the methode.

### Repro steps

The methode' s code:
```csharp
 [FunctionName(nameof(CheckLicense))]
  public async Task<IActionResult> CheckLicense([HttpTrigger(AuthorizationLevel.Anonymous,"post", Route = "licenses/check")]
  HttpRequest request,
  ILogger log)
  {
            try
            {
                //check authorization
                     ....
           }
            catch (Exception e)
            {
                log.LogError(...");
                return .....
            }
}
```

### Expected behavior

Sonarcloud should detect the try catch block.

### Actual behavior

An issues for S6421 is shown

### Related information

* C# 9
* Visual Studio 2022

**Comment by mary-georgiou-sonarsource** [comment]
Hello @stefansta1 and thank you for reporting this.
I confirm it's a false positive.
